src/main/java/uk/gov/companieshouse/web/accounts/controller/govuk/GovukCorporationTaxController.java

```diff
@@ -20,14 +20,13 @@
 @RequestMapping("/accounts/corporation-tax")
 public class GovukCorporationTaxController extends BaseController {
 
-    @Value("${gov-uk-full-accounts.uri}")
+    @Value("${gov-uk-file-your-accounts.uri}")
```

Comment:
Since this has changed it may also be worth changing the name of the variable too?

Reply:
💯